### PR TITLE
Added Featureform in the list

### LINKED ISSUE #771: Add Featureform to the Hacktoberfest Swag List
[Featureform](https://www.featureform.com/hacktoberfest-2023) is participating in the Hacktoberfest 2023!

All accepted submissions will receive a commemorative swag package and entry to a $500 raffle!

This issue is open for anyone who wants to contribute to this repo.

Please read the [contributing guideline](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) to understand the formatting and the information expected from the announcement for your PR to be successfully merged.

### PR DESCRIPTION
**Pull request by vaibhav-009**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

Fixes #771

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -223,6 +223,14 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - [Gain points via Code contributions](https://github.com/formbricks/formbricks/issues)
     - [Gain points via Social Media/Design/Documentation](https://formbricks.notion.site/FormTribe-Side-Quests-4ab3b294cfa04e94b77dfddd66378ea2)
 
+#### **Featureform**
+
+- **Swag**: Accepted PRs will get a Featureform Swag package and will be entered for a raffle for a $500 gift card
+- **Requirements**: One merged PR or tutorial submission.
```
Comment (vimode) [inline docs/list.md:229]: Can you please be more briefly specific about the requirements based on the [Contribution Instructions](https://github.com/featureform/hacktoberfest/blob/main/README.md)?

**Comment by vaibhav-009** [comment]
@vimode updated it. 
please take a look.

**Comment by crweiner** [comment]
Hi @vaibhav-009! Can you please also make sure the Featureform is in the correct alphabetical spot in the `F` section? Thank you!

**Comment by vaibhav-009** [comment]
Hey @crweiner fixed the alphabetical order

**Comment by crweiner** [comment]
Hi again @vaibhav-009, and thank you for another PR to add an organization to the Swag List! I appreciate you making the changes @vimode and I suggested. Let's get Featureform up on the list ✅ 

thank you

**Pull request merged**